client/hotrod-client/src/test/java/org/infinispan/client/hotrod/HotRodClientNearCacheJmxTest.java

```diff
@@ -46,23 +46,19 @@ protected void setup() throws Exception {
       rcms = new RemoteCacheManager[2];
       remoteCaches = new RemoteCache[2];
       for (int i = 0; i < 2; i++) {
-         rcms[i] = addRemoteCacheManager();
+         org.infinispan.client.hotrod.configuration.ConfigurationBuilder clientBuilder =
+               HotRodClientTestingUtil.newRemoteConfigurationBuilder();
+         clientBuilder.addServer().host("localhost").port(hotrodServer.getPort());
+         clientBuilder.nearCache().mode(NearCacheMode.INVALIDATED).maxEntries(100);
+         clientBuilder.statistics().enable()
+               .jmxEnable()
+               .jmxDomain(MethodHandles.lookup().lookupClass().getSimpleName() + i)
+               .mBeanServerLookup(mBeanServerLookup);
+         rcms[i] = new RemoteCacheManager(clientBuilder.build());
```

Comment:
I liked the separate method :)

Reply:
The method surely does not ```add*``` anything. Create maybe